pkgs/development/python-modules/catppuccin-catwalk/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pillow,
+}:
+buildPythonPackage rec {
```

```suggestion
buildPythonApplication rec {
```
I don't think this is a python library.
Would you please use `buildPythonApplication` here?